content/sensu-go/5.20/reference/entities.md

```diff
@@ -38,6 +38,19 @@ All [commercial features][9] are available for free in the packaged Sensu Go dis
 If your Sensu instance includes more than 100 entities, [contact us][10] to learn how to upgrade your installation and increase your limit.
 See [the announcement on our blog][11] for more information about our usage policy.
 
+Commercial licenses may include an entity limit and entity class limits:
+
+- Entity limit: the maximum number of entities of all classes your license includes. Both agent and proxy entities count toward the overall entity limit.
+- Entity class limit: the maximum number of a specific class of entities (e.g. agent or proxy) that your license includes.
+
+For example, if your license has an entity limit of 10,000 and an agent entity class limit of 3,000, you cannot run more than 10,000 entities (agent and proxy) total.
+At the same time, you cannot run more than 3,000 agents.
+If you use only 1,500 agent entities, you can have 8,500 proxy entities before you reach the overall entity limit of 10,000.
+
+Use sensuctl or the license API to [view your overall entity count and limit][29].
+Your entity class limits are listed in your YAML/JSON Sensu license.
+To find your entity class counts, query your store or sensuctl with a tool like [jq][30].
```

Comment:
IMO we shouldn't document workarounds for features that don't exist yet. I think the issue history is sufficient enough for this. Might be best to remove these lines.